third_party/rust/encoding_rs/v0_8/README.chromium

```diff
@@ -5,5 +5,5 @@ Version: 0.8.32
 Security Critical: yes
 Shipped: yes
 License: Apache 2.0 | BSD 3-Clause
```

Comment:
SPDX syntax for union of licenses: https://spdx.dev/learn/handling-license-info/#how
```suggestion
License: Apache-2.0 OR BSD-3-Clause
```